### ISSUE TITLE
sed: file commands.sed line 7: Invalid preceding regular expression

### ISSUE THREAD

**Issue by Codelica** [chipotle/reql-dash-build#1]
Thanks for this!  I desperately want to build them. :)

Getting the same sed error when running the script though, in both Centos 6 (GNU sed version 4.2.1) and Ubuntu 16.04 (GNU sed 4.2.2):

`sed: file commands.sed line 7: Invalid preceding regular expression`

Error is repeated (presumably) for every doc, and result is a build with all the commands but information page is empty for each in Dash.

Any suggestions?

**Comment by Codelica** [comment]
Just to follow up here.  It appears GNU sed would like:

`sed --posix -f commands.sed $doc | pandoc -o \`

vs.

`sed -E -f commands.sed $doc | pandoc -o \`

Also it appears the image assets have been moved within the Rethink docs repo.(?)  In order for the script to find them from my clone (done today) it needed to be:

`cp $docroot/_jekyll/_images/api_illustrations/* \`

vs.

`cp $docroot/assets/images/docs/api_illustrations/* \`

Although I can't say I've seen an image in them yet. :) 

Anyway, THANKS for this !!  👍

**Comment by chipotle** [comment]
I saw this earlier today but didn't get a chance to look into it. I figured there was a good chance this had something to do with BSD sed vs. GNU sed.

I think you're right about the images moving—I'll update that, and put in a comment about GNU sed, at least. (And see if there are actually images showing up, which there were _supposed_ to be occasionally...)

One question: is there a Dash clone for Linux/BSD? I actually hadn't expected anyone to build this on anything but OS X. :)
